test_helpers/example_test.go

```diff
@@ -0,0 +1,24 @@
+package test_helpers_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+func TestExampleMockDoer(t *testing.T) {
+	mockDoer := test_helpers.NewMockDoer(t,
+		test_helpers.NewMockResponse(t, []interface{}{"some data"}),
+		fmt.Errorf("some error"),
+	)
+
+	data, err := mockDoer.Do(&test_helpers.MockRequest{}).Get()
+	assert.NoError(t, err)
+	assert.Equal(t, []interface{}{"some data"}, data)
+
+	data, err = mockDoer.Do(&test_helpers.MockRequest{}).Get()
+	assert.EqualError(t, err, "some error")
+	assert.Nil(t, data)
```

Comment:
Let's make some improvements in the example:
1. Add a real requests to show that a response does not depend on a request.
2. Add `GetTyped()` call too to show that it works too.
```suggestion
	data, err := mockDoer.Do(tarantool.NewPingRequest()).Get()
	assert.NoError(t, err)
	assert.Equal(t, []interface{}{"some data"}, data)

	data, err = mockDoer.Do(tarantool.NewSelectRequest("foo")).Get()
	assert.EqualError(t, err, "some error")
	assert.Nil(t, data)
```